recipes/pystack/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pystack" %}
```

Comment:
Can we raise an issue and point out this naming conflict to them? Maybe they would be willing to rename given the confusion this is causing.